$OpenBSD$

Pickup the right header when both e2fsprogs and ossp-uuid are installed.

--- src/sharp/uuid.hpp.orig	Mon Sep 26 15:01:02 2011
+++ src/sharp/uuid.hpp	Mon Sep 26 15:02:25 2011
@@ -28,7 +28,7 @@
 #ifndef __SHARP_UUID_HPP_
 #define __SHARP_UUID_HPP_
 
-#include <uuid.h>
+#include <uuid/uuid.h>
 #include <string>
 
 namespace sharp {
